src/assets/templates/typescript/src/steps/user-field-equals.ts

```diff
@@ -40,6 +44,11 @@ export class UserFieldEqualsStep extends BaseStep implements StepInterface {
     field: 'field',
     type: FieldDefinition.Type.STRING,
     description: 'Field name to check',
+  }, {
+    field: 'operator',
+    type: FieldDefinition.Type.STRING,
+    optionality: FieldDefinition.Optionality.OPTIONAL,
```

Comment:
- [ ] Would recommend keeping this set to required for all new steps going forward.  The optionality was just for backward compatibility.

Reply:
Good point!